.ci/.jenkins_nodejs.yml

```diff
@@ -7,5 +7,3 @@ NODEJS_VERSION:
   - "10.0"
   - "8"
   - "8.6"
-  - "6"
```

Comment:
1. Can we keep this one? I specifically meant only dropping "6.0" -- i.e. testing with the first 6.x release of node.js.
2. Also, please change the title of this issue (and the commit message when merging) to reflect that this is just dropping "6.0" testing (because it fails for an unknown reason and it isn't worth the time to dig into why right now).

Reply:
Sure thing. Done.